chart/scripts/post-delete-hook.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+
+echo "Preparing for uninstalling apps left by Rancher in the following namespaces:" "$namespaces"
+for namespace in $namespaces; do
```

Comment:
Does namespaces actually get inherited from the env here ? I'm surprised its in the shells context.

Reply:
yes, the script gets the environment variable from the container.